client/src/components/ElementHistoryViewer/ElementHistoryViewer.js

```diff
@@ -0,0 +1,14 @@
+// import React from 'react';
+import { compose } from 'redux';
+import readOneBlockQuery from 'state/history/readOneBlockQuery';
+
+/**
+ * Transformer function which binds the readOneBlockQuery GraphQL query to the HistoryViewer
+ * component.
+ */
+const withElementsHistoryViewer = (HistoryViewer) => compose(
+  readOneBlockQuery
+)(HistoryViewer);
+
```

Comment:
I might be missing something, but I don't think this function is necessary. `readOneBlockQuery` exports an HOC function, expecting a Component as an argument. This should suffice:

```js
updater.component(
  'HistoryViewer.Form_ItemEditForm',
  readOneBlockQuery,
  'ElementHistoryViewer'
);
```

Reply:
I think you're right